tests/security/swtpm/swtpm_env_setup.pm

```diff
@@ -5,18 +5,38 @@
 #          install required packages and download the pre-installed
 #          OS images for later tests, prepare a script for remote
 #          ssh login and execution
-# Maintainer: rfan1 <richard.fan@suse.com>
+# Maintainer: rfan1 <richard.fan@suse.com> Starry Wang <starry.wang@suse.com>
 # Tags: poo#81256, tc#1768671
```

Comment:
@STARRY-S 
I think you forgot to put poo here. #100512

Reply:
Done, thanks for the comments.